src/symbol/glyph_source.js

```diff
@@ -50,22 +55,33 @@ class GlyphSource {
         // the number of pixels the sdf bitmaps are padded by
         const buffer = 3;
 
-        const missing = {};
+        const missingRanges = {};
         let remaining = 0;
 
         const getGlyph = (glyphID) => {
             const range = Math.floor(glyphID / 256);
+            if (this.localIdeographFontFamily &&
+                (isChar['CJK Unified Ideographs'](glyphID) ||
+                 isChar['Hangul Syllables'](glyphID))) {
```

Comment:
@ChrisLoer made a proof of concept of this:

![screenshot 2017-06-30 14 55 25](https://user-images.githubusercontent.com/1231218/27757395-c7f18fe2-5db5-11e7-8fbf-1f0509bd0b6f.png)

> it was pretty easy to render http://3.decodeunicode.org/en/u+2B776 just by find/replacing “charCode” with “codePoint” and using TinySDF for every glyph over 65535.

Incidentally, I did some additional research about the prevalence of affected characters in the Mapbox Streets source: https://github.com/mapbox/mapbox-gl-js/issues/4001#issuecomment-312393003.